streams/src/main/java/org/apache/kafka/streams/processor/TaskId.java

```diff
@@ -25,11 +25,13 @@
 import java.nio.ByteBuffer;
 
 /**
- * The task id representation composed as topic group id plus the assigned partition id.
+ * The task ID representation composed as topic group ID plus the assigned partition ID.
```

Comment:
According to the implementation is it `group-id-partition-id`. But why would this be important?